lessons/Projects/project-design-a-survey/langs/en-us/index.adoc

```diff
@@ -61,18 +56,26 @@ Now that you know what questions your team wants to ask, it's time to actually c
 
 @lesson-instruction{
 With your team, create a _first draft_ of the survey by opening a new google form and entering:
+
 * The title of the survey
+
 * A description of what the survey is for and how responses will be used
-* At least 8 good questions, copied from your @opt-printable-exercise{survey-brainstorm.adoc}.}
 
-Once a rough draft has been completed, students are ready to make improvements to the survey so that it yeilds the cleanest results possible!
+* At least 8 good questions, copied from your @handout{survey-brainstorm.adoc}.}
+
+Once a rough draft has been completed, students are ready to make improvements to the survey so that it yeilds the cleanest results possible.
 
 @lesson-instruction{
 Read through the survey with your team, considering each of the questions one by one, and making improvements along the way.
+
```

Comment:
@retabak Why do these need to have extra blank spaces?

Reply:
@schanzer - not sure what you mean? 

if I remove the blank lines (is that what you're talking about?), this is what happens:

![image](https://github.com/bootstrapworld/curriculum/assets/87144723/4ac9d0d6-8061-4e9c-aed7-65043645a62c)

Reply:
No, that's what happens if you get rid of the _FIRST_ blank line. In asciidoc, lists always need a blank line before they start.

The rest of the blanks are unecessary